ecs/wrapper.go

```diff
@@ -715,11 +716,18 @@ func CreateScheduledTask(cluster, service, taskSuffix, scheduleExpression, comma
 		target.RoleArn = role.Arn
 	}
 	if command != "" {
+		reader := csv.NewReader(strings.NewReader(command))
+		reader.Comma = ' '
+		commands, err := reader.Read()
+		if err != nil {
+			return err
```

Comment:
I think it would be worthwhile to wrap the error here to make it more clear why the command is failing, something like `return fmt.Errorf("command syntax invalid: %v", err)`